src/components/GlobalHeader/index.js

```diff
@@ -1,60 +1,44 @@
-import { Component } from 'react';
-import { routerRedux, Link } from 'dva/router';
-import { Icon, Divider, Tooltip, Alert, Badge, Tag } from 'antd';
-import { Dropdown, DropdownItem, DropdownToggle, Avatar } from '@patternfly/react-core';
-import Debounce from 'lodash-decorators/debounce';
+import React, { Component } from 'react';
+import { routerRedux } from 'dva/router';
+import {
+  PageHeader,
+  PageHeaderTools,
```

Comment:
Is some error as follows thrown when importing `PageHeaderTools` in your system? I am pretty sure this warning is thrown during import of one of these: PageHeaderTools, PageHeaderToolsGroup.
```
Warning: React.createElement: type is invalid -- expected a string 
(for built-in components) or a class/function (for composite components)
 but got: undefined. You likely forgot to export your component from
 the file it's defined in, or you might have mixed up default and named imports.
```

Reply:
I wasn't able to reproduce the issue on my local build; however, I would try upgrade your Patternfly React dependency as it might be referencing a previous version of the library.

Reply:
@gurbirkalsi My dependencies are at
```
"@patternfly/react-charts": "^5.3.19",
"@patternfly/react-core": "^3.153.2",
"@patternfly/react-icons": "^3.15.16",
"@patternfly/react-table": "^2.28.39",
```